packages/pigeon/platform_tests/shared_test_plugin_code/lib/test_types.dart

```diff
@@ -183,40 +233,141 @@ void compareAllClassesWrapper(
   );
   compareAllTypes(wrapperOne.allTypes, wrapperTwo.allTypes);
 
-  for (int i = 0; i < (wrapperOne.classList.length); i++) {
-    compareAllTypes(wrapperOne.classList[i], wrapperTwo.classList[i]);
-  }
-  if (wrapperOne.nullableClassList != null &&
-      wrapperTwo.nullableClassList != null) {
-    for (int i = 0; i < (wrapperOne.nullableClassList!.length); i++) {
-      compareAllNullableTypesWithoutRecursion(
-          wrapperOne.nullableClassList![i], wrapperTwo.nullableClassList![i]);
-    }
-  } else {
-    expect(wrapperOne.nullableClassList != null,
-        wrapperTwo.nullableClassList != null);
-  }
-  final Iterable<int?> wrapperOneKeys = wrapperOne.classMap.keys;
-  final Iterable<int?> wrapperTwoKeys = wrapperTwo.classMap.keys;
-  expect(wrapperOneKeys.length, wrapperTwoKeys.length);
-  for (final int? key in wrapperOneKeys) {
-    compareAllTypes(wrapperOne.classMap[key], wrapperTwo.classMap[key]);
-  }
-  if (wrapperOne.nullableClassMap != null &&
-      wrapperTwo.nullableClassMap != null) {
-    final Iterable<int?> wrapperOneKeys = wrapperOne.nullableClassMap!.keys;
-    final Iterable<int?> wrapperTwoKeys = wrapperTwo.nullableClassMap!.keys;
-    expect(wrapperOneKeys.length, wrapperTwoKeys.length);
-    for (final int? key in wrapperOneKeys) {
-      compareAllNullableTypesWithoutRecursion(
-          wrapperOne.nullableClassMap![key], wrapperTwo.nullableClassMap![key]);
-    }
-  } else {
-    expect(wrapperOne.nullableClassMap != null,
-        wrapperTwo.nullableClassMap != null);
-  }
+  // for (int i = 0; i < (allTypesOne.classList.length); i++) {
+  //   compareAllTypes(
+  //       allTypesOne.classList[i], allTypesTwo.classList[i]);
+  // }
+  //
+  // classMap: check each key and value, call each compare method for every class
+  //
+  // for (int i = 0; i < (allTypesOne.nullableClassList.length); i++) {
+  //   compareAllNullableTypesWithoutRecursion(
+  //       allTypesOne.nullableClassList[i], allTypesTwo.nullableClassList[i]);
+  // }
```

Comment:
We definitely shouldn't be replacing existing tests with commented out tests.

Reply:
bad merge :(